### PR TITLE
Client dashboard

### PR DESCRIPTION
**Pull request by linusjig**
add redirection paths

### CODE REVIEW

File: app/controllers/client/invoices_controller.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def authorize
   end
 
   def set_invoice
-    @invoice = current_user.invoices.find(params[:id])
+    @invoice = current_user.client.invoices.find(params[:id])
```
Comment (cosydney) [inline app/controllers/client/invoices_controller.rb:27]: yes this one is a bitch it got me too

File: app/controllers/registrations_controller.rb [hunk 2]
```diff
@@ -4,10 +4,6 @@ class RegistrationsController < Devise::RegistrationsController
    # To redirct users after sign-up to edit
 
   def after_sign_up_path_for(user)
-    if current_user.freelancer
-      edit_freelancer_path(user.freelancer.id)
-    else
-      edit_client_path(user.client.id)
-    end
+    edit_freelancer_path(user.freelancer.id)
```
Comment (cosydney) [inline app/controllers/registrations_controller.rb:7]: does this mean the client gets to edit freelancers stuff ?

File: app/models/freelancer.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 class Freelancer < ActiveRecord::Base
-  belongs_to :user, dependent: :destroy
+  belongs_to :user
```
Comment (cosydney) [inline app/models/freelancer.rb:2]: why is the dependent destroy taken off ?

File: app/controllers/registrations_controller.rb (hunk 2 above)
Comment (linusjig) [reply app/controllers/registrations_controller.rb:7]: nope, the invitable is not actually a sign-up, but an after accept invitation. So only the people that sign-up, are freelancers and can edit their freelancers profile

File: app/models/freelancer.rb (hunk 3 above)
Comment (linusjig) [reply app/models/freelancer.rb:2]: That was Cyrille helping me debug the problem with the client freelancer creation, it was messing with the code

File: app/views/client/invoices/index.html.erb [hunk 4]
```diff
@@ -1,18 +1,39 @@
...
-<%= link_to "Create Invoice", new_invoice_path %>
-<div>
-  <%= link_to "Dashboard", dashboard_path %>
-</div>
-<div>
-<%= link_to "Freelancer settings", edit_freelancer_path %>
-</div>
+<% current_user.client.invoices.each do |invoice| %>
+  <%= link_to invoice_path(invoice) do %>
+    <div class="invoice-wrapper" id="<%= invoice.id %>">
```
Comment (cosydney) [inline app/views/client/invoices/index.html.erb:3]: I'm curious to know how you will use invoice.id selector !

File: app/controllers/registrations_controller.rb (hunk 2 above)
Comment (cosydney) [reply app/controllers/registrations_controller.rb:7]: right thank you

**Pull request merged**